navigation/BottomTabNavigation.jsx

```diff
@@ -9,7 +8,8 @@ import ProfileIcon from '../assets/icons/Profile.svg';
 import MessagesIcon from '../assets/icons/Messages.svg';
 
 import StackNavigation from './StackNavigation';
-import MessagesList from '../screens/Messages/MessagesList';
+// import MessagesList from '../screens/Messages/MessagesList';
```

Comment:
remove comment if you don't need it. If you need add some explanation as a c omment

Reply:
Removed comment.